docs/0.4.0-alpha/advanced_syntax/builtins.md

```diff
@@ -18,6 +18,44 @@ Transpile to `echo` which prints text to the console, requires a `Text` paramete
 echo "Hello World!"
 ```
 
+## Len
+
+Transpile to `echo "${#TEXT}"` for a `Text` parameter:
+
+```ab
+echo len("Jackdaws love my big sphinx of quartz")
+```
+
+Transpile to `echo "${#ARRAY[@]}"` for an `Array` parameter:
+
+```ab
+echo len(["one", "two", "three", "four", "five"])
+```
```

Comment:
Let's give the developer more context:

```suggestion

This keyword calculates and returns the length of a given `Text` value, measured in ASCII characters, or the length of an array `[]`. The result is returned as a `Num` type.

For a `Text` parameter, the expression is transpiled internally to the Bash command echo "${#TEXT}". For an `Array` (`[]`) parameter, it is transpiled to echo "${#ARRAY[@]}".

```ab
echo len("Jackdaws love my big sphinx of quartz")
// Returns 37

echo len(["one", "two", "three", "four", "five"])
// Returns 5
```

Reply:
@hdwalters I thought that we could also add information of what does this built-in do and what type of value does it return